levels.go

```diff
@@ -860,14 +860,33 @@ func (s *levelsController) get(key []byte) (y.ValueStruct, error) {
 		if err != nil {
 			return y.ValueStruct{}, errors.Wrapf(err, "get key: %q", key)
 		}
-		if vs.Value == nil && vs.Meta == 0 {
-			continue
+		if vs.Valid() {
+			return vs, nil
 		}
-		return vs, nil
 	}
 	return y.ValueStruct{}, nil
 }
 
+func (s *levelsController) multiGet(pairs []keyValuePair) error {
+	for _, h := range s.levels {
+		for i := range pairs {
+			pair := &pairs[i]
+			if pair.found {
+				continue
+			}
+			val, err := h.get(pair.key) // Calls h.RLock() and h.RUnlock().
```

Comment:
Can we reduce calling of RLock and RUnlock

Reply:
We can move the `getTablesForKey` and `decrTableRef` out of `h.get`, then move `RLock` out of `getTablesForKey`, but after I do it, I found the code become much harder to read.
So I prefer not to do so.

Reply:
I'll try another way to do it.